baus/subsidies.py

```diff
@@ -191,6 +195,28 @@ def lump_sum_accounts(policy, year, buildings, coffer,
         coffer[acct["name"]].add_transaction(amt, subaccount=1,
                                              metadata=metadata)
 
+@orca.step()
+def office_lump_sum_accounts(policy, year, buildings, coffer,
+                             summary, years_per_iter, scenario):
+
+    s = policy["acct_settings"]["office_lump_sum_accounts"]
+
+    for key, acct in s.items():
+        if scenario not in acct["enable_in_scenarios"]:
+            continue
+
+        amt = float(acct["total_amount"])
+
+        amt *= years_per_iter
+
+        metadata = {
+            "description": "%s subsidies" % acct["name"],
+            "year": year
+        }
+        # the subaccount is meaningless here (it's a regional account) -
+        # but the subaccount number is referred to below
+        coffer[acct["name"]].add_transaction(amt, subaccount="regional",
+                                             metadata=metadata)
```

Comment:
Leaving another comment for myself to look at how subaccount is used. It looks like the residential version uses a number but also like subaccount doesn't get used anyway.

Reply:
It's not. The policy.yaml doesn't setup sub-accounts under account. I imagine it would be more like the vmt strategy - we essentially have 2 sub-accounts: com_for_com and com_for_res; fees collected are put into these subaccounts, and then used to subsidize office or residential.

Reply:
Seems like this line [here](https://github.com/BayAreaMetro/bayarea_urbansim/blob/fb_subsidize_office_dev_v1/baus/subsidies.py#L876) is what makes "1" work [here](https://github.com/BayAreaMetro/bayarea_urbansim/blob/fb_subsidize_office_dev_v1/baus/subsidies.py#L882). 

Looks like your model uses it for the [coffer](https://github.com/BayAreaMetro/bayarea_urbansim/blob/fb_subsidize_office_dev_v1/baus/subsidies.py#L653), but doesn't need to filter `feasibility` by it.

Reply:
We can revise it to be able to filter by subaccount, so we will have only one account with multiple subaccounts. This is the alternative approach I was talking about in the Asana task. However, I think it's relatively easy to make changes to these function, but harder to set it up correctly in policy.yaml.